Shared/Samples/Add custom dynamic entity data source/AddCustomDynamicEntityDataSourceView.Vessel.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extension AddCustomDynamicEntityDataSourceView {
+    /// A marine vessel that can be decoded from the vessel JSON.
+    struct Vessel {
+        /// The location of the vessel.
+        let geometry: (x: Double, y: Double)
```

Comment:
Why was this changed to a tuple?

Reply:
I did that so the decodable `Geometry` type could remain an implementation detail.

Reply:
It wasn't an implementation detail before because it was actually being used outside of the implementation. So there is no reason that I can see to make it a private implementation detail. I only suggested that for `Attributes` because the individual attributes themselves weren't being used outside of the implementation. But as I got further in my review, I realized that converting the attributes to a dictionary isn't even necessary. I probably should have gone back and revised my earlier comments.